docs/manual/developer_guide.adoc

```diff
@@ -1600,11 +1600,14 @@ audit_rules_usergroup_modification::
 ** *path* - path that should be part of the audit rule as a value of `-w` argument, eg. `/etc/group`.
 * Languages: Ansible, Bash, OVAL
 
-bls_bootloader_option::
-* Checks kernel command line arguments in BLS-compatible (Boot Loader Specification) boot loader configuration.
+argument_in_file::
```

Comment:
I suggest `argument_value_in_line` as a more descriptive and accurate name.

Reply:
Or `parameter_value_in_line`.

Reply:
Okay, `argument_value_in_line` it is